src/procstat.c

```diff
@@ -363,7 +353,7 @@ static void fuse_lookup(fuse_req_t req, fuse_ino_t parent_inode, const char *nam
 	item->refcnt++;
```

Comment:
This modified the refcnt while holding the parent's lock, which is not consistent with fuse_forget() that does it under item's lock.